airbyte-integrations/connectors/source-orbit/source_orbit/schemas/members.json

```diff
@@ -0,0 +1,163 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "data": {
```

Comment:
@marcosmarxm The orbit API returns an extra "data" here, is there a way to bypass this?